services/transformer/handler.go

```diff
@@ -0,0 +1,115 @@
+package transformer
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/tidwall/gjson"
+
+	"github.com/rudderlabs/rudder-go-kit/config"
+	"github.com/rudderlabs/rudder-go-kit/logger"
+	"github.com/rudderlabs/rudder-server/utils/httputil"
+)
+
+type featuresService struct {
+	logger   logger.Logger
+	waitChan chan struct{}
+	config   FeaturesServiceConfig
+	features json.RawMessage
+}
+
+func (t *featuresService) SourceTransformerVersion() string {
+	if gjson.Get(string(t.Get()), "supportSourceTransformV1").Bool() {
+		return V1_SOURCE_TRANSFORMER_VERSION
+	}
+
+	return V0_SOURCE_TRANSFORMER_VERSION
+}
+
+func (t *featuresService) Get() json.RawMessage {
+	if t.features == nil {
+		return json.RawMessage(defaultTransformerFeatures)
+	}
+
+	return t.features
+}
+
+func (t *featuresService) Wait() chan struct{} {
+	return t.waitChan
+}
+
+func (t *featuresService) init(ctx context.Context) {
+	go t.syncTransformerFeatureJson(ctx)
+}
```

Comment:
This seems to be redundant as a method, we may simply call `rruntime.Go(func() {t.syncTransformerFeatureJson(ctx)})` in `NewFeaturesService`